src/app/components/privacy-policy/privacy-policy.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
```

Comment:
License is missing & not documentation for component

Reply:
Added, thanks!